app/controllers/projects_controller.rb

```diff
@@ -294,7 +294,7 @@ def head_of_project_users_list
   end
 
   def assigned_users_list
-    users = User.where(id: @project.user_assignments.select(:user_id)).order('full_name ASC')
+    users = User.where(id: @project.user_assignments.select(:user_id)).search(false, params[:query]).order('full_name ASC')
```

Comment:
@aignatov-bio  Why we can't use here this simpler code?

 @project.users.search(false, params[:query]).order(:full_name)